src/Dfe.EarlyYearsQualification.Content/Services/ContentfulContentService.cs

```diff
@@ -208,4 +208,46 @@ public async Task<List<NavigationLink>> GetNavigationLinks()
         var checkAdditionalRequirementsAnswerPage = checkAdditionalRequirementsAnswerPageEntities.First();
         return checkAdditionalRequirementsAnswerPage;
     }
+    
+    public async Task<CannotFindQualificationPage?> GetCannotFindQualificationPage(int level, int startMonth, int startYear)
+    {
+        var cannotFindQualificationPageType = ContentTypeLookup[typeof(CannotFindQualificationPage)];
+        var queryBuilder = new QueryBuilder<CannotFindQualificationPage>().ContentTypeIs(cannotFindQualificationPageType)
+                                                                          .Include(2)
+                                                                          .FieldEquals("fields.level", level.ToString());
+        
+        var cannotFindQualificationPages = await GetEntriesByType(queryBuilder);
+        if (cannotFindQualificationPages is null || !cannotFindQualificationPages.Any())
+        {
+            Logger.LogWarning("No 'cannot find qualification' page entries returned");
+            return default;
+        }
+
+        var filteredCannotFindQualificationPages =
+            FilterCannotFindQualificationPagesByDate(startMonth, startYear, cannotFindQualificationPages.ToList());
+
+        if (filteredCannotFindQualificationPages.Count != 0) return filteredCannotFindQualificationPages.First();

```

Comment:
Suggestion (SonarCloud): `filteredCannotFindQualificationPages[0]` (we know it's a list) instead of `.First()`

Reply:
Updated now